### PR TITLE
Implement migration integration test

### PR DESCRIPTION
**Pull request by tarrencev**

### CODE REVIEW

**Comment by ponderingdemocritus** [comment]
We should include the CORS update otherwise clients cannot connect

**Comment by tarrencev** [comment]
Do you mind opening that as a separate PR?

**Comment by ponderingdemocritus** [comment]
yes! ill wait till this is merged

**Review by kariy — APPROVED** [review]

File: examples/target/CACHEDIR.TAG
Comment (kariy) [inline examples/target/CACHEDIR.TAG:1]: Is this file generated by `scarb` ?

**Pull request merged**